runtime/src/enclave_rpc/client.rs

```diff
@@ -490,7 +465,18 @@ impl RpcClient {
             args: cbor::to_value(args),
         };
 
-        let (pfid, inner) = match self.execute_call(request, kind).await {
+        // We can only retry transport errors as for those we are sure that the request was not
```

Comment:
But do we retry only transport errors? I don't see any checks for `RpcClientError`. :thinking:

Reply:
What kind of other errors will return an `Err`, I think all should be transport errors, but let me check.

Reply:
I didn't check that. But the comment is confusing, as I would except that we check that there is a transport error.